modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,54 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run many commands in parallel on a large number of clients.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting features to control the load in your environment.
+
+These features are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+
+
+== Batch Size
+
+Actions are processed in batches.
+This parameter sets the maximum number of clients that can execute a single action in each batch.
+
+Choosing a lower number can prevent system thrashing, but might reduce overall performance as parallelization is reduced.
+
+Adjust the [systemitem]``java.salt_batch_size`` parameter.
+Defaults to 200.
+
+
+
+== Batch Delay
+
+This parameter sets the amount of time, in seconds, to wait after a batch is processed before beginning to process the next batch.
+The first batch will process the number of actions determined in the batch size parameter, and subsequent batches will process fewer actions.
```

Comment:
I find this part a little bit confusing, given that we are talking about actions, but in reality what happens is that we talk about minions. 
The first batch actually will process whatever we say in a "batch size" amount of minions, and subsequent batches will process likely fewer minions than the first one, but at the end we will always have, at any given point in time, the same number of minions in execution in parallel.